cost/scheduled_report_markUpsDowns/scheduled_report_markUpsDowns.pt

```diff
@@ -0,0 +1,429 @@
+name "Scheduled Report with markups"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy allows you to set up scheduled reports that will provide summaries of cloud cost across all resources in the billing centers with a markup or markdown % for the last month. See the [README](https://github.com/rightscale/policy_templates/tree/master/Cost/scheduled_report_markupsDowns/readme.md) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
The README link will fail because the readme.md file is not the proper case.  You can simply remove **readme.md** to solve this.